include/spirv/unified1/spirv.core.grammar.json

```
@@ -14909,6 +14929,13 @@
           "extensions" : [ "SPV_INTEL_kernel_attributes" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "FPGALatencyControlINTEL",
+          "value" : 6171,
+          "extensions" : [ "SPV_INTEL_fpga_latency_control" ],
+          "extensions" : [ "SPV_INTEL_kernel_attributes" ],
```

I don't see the interaction with `SPV_INTEL_kernel_attributes` in the spec.  Is this intended?

If it is intended I think you want a slightly different syntax here:

```suggestion
          "extensions" : [ "SPV_INTEL_fpga_latency_control", "SPV_INTEL_kernel_attributes" ],
```

---

Ah...Good catch. This was not intended. Resolved it. Thanks